pages/en/contrib/pages.md

```diff
@@ -23,6 +23,8 @@ Under English, the sub-folders are:
 - `contrib` - Information on contributing to the project, published at [http://loopback.io/doc/en/contrib/](http://loopback.io/doc/en/contrib/).
 - `community` - Area where community members can put information about their LoopBack-related projects.
 
+However, for LoopBack 4 (`lb4`), the documentation is sourced from the `/docs/site` folder in the `loopback-next` repository. 
```

Comment:
Is the `lb4` needed here? Feel free to keep it, but I think it's better to remove it.